frontend/README.md

```diff
@@ -40,3 +40,63 @@ yarn clean # clear previous build artifacts
 2. Fill in the `.env` file with your Infura ID. You only need the Portis and Fortmatic API keys if you plan on using those wallets
 3. Install dependencies with `yarn`
 4. Build for development with `yarn run dev`
+
+## Internationalization
```

Comment:
*(This is an arbitrary comment location)*

When I navigate to the FAQ, change the language to Chinese, then navigate back to the homescreen, the language picker becomes "corrupted." It now shows the language as English, with two choices listed, both English.

![lang-switch-issue](https://user-images.githubusercontent.com/1479802/168116794-3b339c37-3b3e-44e7-a718-87ea774460fc.gif)